solutions/standard/override.json

```diff
@@ -845,33 +846,33 @@
       },
       {
          "boot_volume_encryption_key_name": null,
-         "image_name": "ibm-zos-2-5-s390x-dev-test-wazi-6",
+         "image_name": "ibm-zos-2-4-s390x-dev-test-wazi-10",
```

Comment:
Why did we change the default image_name value to a really old 2.4 image ?

Reply:
Updated